### PR TITLE
(feat) O3-3899: Tweak photo picker in the registration form

### PR DESCRIPTION
**Pull request by Twiineenock**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

This PR resolves several issues with the photo picker as described in detail within [this ticket](https://openmrs.atlassian.net/browse/O3-3899). The updates are based on the [design outlined here](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/60d59b0642e296b9bd541101), aimed at improving user experience and functionality.

- [ ] Enabled users to add or change an image directly in place.
- [ ] Integrated upload files and webcam options within the same context for enhanced flexibility.
- [ ] Updated the edit button functionality to toggle between add mode and change mode.

## Screenshots

**Addition mode(No image to display)**
add-mode

**Edit mode(changing an image)**
edit-mode-2

## Related Issue

[https://openmrs.atlassian.net/browse/O3-3899](https://openmrs.atlassian.net/browse/O3-3899)

## Other

**Screancast to demonstrate the workflow**

[photo-picker-2.webm](https://github.com/user-attachments/assets/ed137f50-a296-4f79-bd2a-235cdece46df)

### CODE REVIEW

**Comment by Twiineenock** [comment]
FYI @usamaidrsk , @vasharma05

**Review by usamaidrsk — CHANGES_REQUESTED** [review]
Thanks @Twiineenock for the work, I have a few suggestions.

I am not sure if this component is only used to capture patient image, I will refer that to @denniskigen or @vasharma05 to confirm.

File: packages/esm-patient-attachments-app/src/camera-media-uploader/camera-media-uploader.component.tsx [hunk 1]
```diff
@@ -115,7 +121,7 @@ const CameraMediaUploadTabs = () => {
 
   return (
     <div className={styles.cameraSection}>
-      <ModalHeader closeModal={closeModal} title={t('addAttachment_title', 'Add Attachment')} />
+      <ModalHeader closeModal={closeModal} title={title ? title : t('addAttachment_title', 'Add Attachment')} />
```
Comment (usamaidrsk) [inline packages/esm-patient-attachments-app/src/camera-media-uploader/camera-media-uploader.component.tsx:124]: You can make this simple as `title || t('addAttachment_title', 'Add Attachment')`

File: packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.scss [hunk 2]
```diff
@@ -1,33 +1,67 @@
...
+.placeholderText {
+  width: 57px;
+  height: 32px;
+  font-family: IBMPlexSans;
+  font-size: 12px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.33;
+  letter-spacing: 0.32px;
```
Comment (usamaidrsk) [inline packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.scss:19]: You may ignore these styles

File: packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.scss [hunk 3]
```diff
@@ -1,33 +1,67 @@
...
+  height: 18px;
+  margin-left: 0.5rem;
+  padding-left: 15px;
+  font-family: IBMPlexSans;
+  font-size: 14px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: 1.29;
+  letter-spacing: 0.16px;
```
Comment (usamaidrsk) [inline packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.scss:49]: You may ignore these too

File: packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.scss [hunk 4]
```diff
@@ -1,33 +1,67 @@
...
+  color: colors.$white-0;
+}
+
+.editButton {
+  display: flex;
+  width: 1rem;
+  height: 1rem;
```
Comment (usamaidrsk) [inline packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.scss:56]: use `layout.$spacing-05`

File: packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.scss [hunk 5]
```diff
@@ -1,33 +1,67 @@
...
-  height: layout.$spacing-10;
-  background-color: #d8d8d8;
+.editImage {
   display: flex;
+  flex-direction: row;
   align-items: center;
-  justify-content: center;
+  justify-content: space-between;
+  width: 120px;
+  height: 48px;
```
Comment (usamaidrsk) [inline packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.scss:33]: use `layout.$spacing-09`

Comment (Twiineenock) [reply packages/esm-patient-attachments-app/src/camera-media-uploader/capture-photo.scss]: Thanks @usamaidrsk! We pushed some commits to with changes.

**Review by denniskigen — APPROVED** [review]
LGTM

**Pull request merged**